examples/azure/terraform-azure-mysqldb-example/main.tf

```diff
@@ -0,0 +1,67 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AN AZURE MySQL Database
+# This is an example of how to deploy an Azure Mysql database.
+# See test/terraform_azure_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+
+# ---------------------------------------------------------------------------------------------------------------------
+# CONFIGURE OUR AZURE CONNECTION
+# ---------------------------------------------------------------------------------------------------------------------
+
+provider "azurerm" {
+  version = "~>2.29.0"
+  features {}
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A RESOURCE GROUP
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_resource_group" "mysql_rg" {
+  name     = "terratest-mysql-${var.postfix}"
+  location = var.location
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AZURE MySQL SERVER
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "random_password" "password" {
```

Comment:
NIT: Add caveat about security in the same manner as https://github.com/gruntwork-io/terratest/blob/master/examples/azure/terraform-azure-vm-example/main.tf#L133